ClemBot.Bot/bot/services/tag_service.py

```diff
@@ -21,9 +23,12 @@ def __init__(self, *, bot):
     @BaseService.Listener(Events.on_guild_message_received)
     async def on_guild_message_received(self, message: discord.Message) -> None:
 
+        #tagprefix = await self.bot.custom_tag_prefix_route.get_custom_tag_prefixes(message.guild.id, raise_on_error=True)
+        tagprefix = await self.get_tag_prefix(self.bot, message=message)
+        tagprefix = tagprefix[0]
         tags_content = []
         tag_found = False
-        pattern = re.compile(f'(^|\\s)[{TAG_PREFIX}](\\w+)')
+        pattern = re.compile(f'(^|\\s)[{tagprefix}](\\w+)')
```

Comment:
```suggestion
        pattern = re.compile(fr'(^|\s)[{re.escape(tagprefix)}](\w+)')
```
This is so we can use regex characters like ^ and * as prefixes as well without breaking things